packages/edit-site/src/components/block-editor/index.js

```diff
@@ -257,8 +254,6 @@ export default function BlockEditor( { setIsInserterOpen } ) {
 							<BlockEditorKeyboardShortcuts.Register />
 							<BackButton />
 							<ResizableEditor
-								// Reinitialize the editor and reset the states when the template changes.
-								key={ templateId }
```

Comment:
If a reset in size is needed, we should be doing it inside the `ResizableEditor` components and not remount the whole tree.